### DISCUSSION TITLE
Qfield wont connect to Arrow GPS.

### DISCUSSION THREAD

**Discussion by Finwhit** [opengisch/QField#3655] [Q&A]
I'm trying to connect my arrow GPS to QField, had no problems with it until this morning. When I try to scan for devices to connect to in the Qfield settings it comes up with a message saying "Scanning failed: Classic Discovery cannot be started" anyone have any ideas why it is doing this?

**Comment by nirvn** [comment]
This is a known issue with devices running Android >= 12. Fixed in QField 2.5.3.

**Reply by Finwhit** [reply]
Ah right this may explain it my phone is running version 2.5.2 of Qfield tho I haven't had this issue before. Any ideas how I can update to version 2.5.3? Only seems to be version 2.5.2 in the playstore and no options to update

**Comment by nirvn** [comment]
@Finwhit , you can manually update using the APKs here:

https://github.com/opengisch/QField/releases/tag/v2.5.3

Otherwise wait 48 hours for play store to catch up :)
